### PR TITLE
Created comment functionality for Events and deleted message feed

### PR DESCRIPTION
**Pull request by Ajide49**
I created a Comment.java class that will hold the data and definition for every comment identity corresponding to an event. I created a couple of functions in Datastore that deal with saving the comments to datastore and created an arraylist of comments of just the given Event UUID. I also made some changes in the EventOneSevlet to display comments and receive the data for them. I have not yet fully tested these new Datastore functions or added any html/js functionality for the comments.

### CODE REVIEW

**Comment by ryanluo** [comment]
Can you make sure you resolve conflicts first?

File: pom.xml [hunk 1]
```diff
@@ -67,6 +58,52 @@ limitations under the License.
...
+
+    <!-- Test Dependencies
+         <dependency>
+             <groupId>junit</groupId>
+             <artifactId>junit</artifactId>
+             <version>4.12</version>
+             <scope>test</scope>
+         </dependency> -->
+
+         <dependency>
```
Comment (kassidywebb) [inline pom.xml]: Formatting still needs to be fixed

File: pom.xml [hunk 2]
```diff
@@ -67,6 +58,52 @@ limitations under the License.
...
+             <version>1.9.60</version>
+             <scope>test</scope>
+         </dependency>
+         <dependency>
+             <groupId>com.google.appengine</groupId>
+             <artifactId>appengine-tools-sdk</artifactId>
+             <version>1.9.60</version>
+             <scope>test</scope>
+         </dependency>
+      <!--   <dependency>
```
Comment (kassidywebb) [inline pom.xml]: Are we keeping this dependency? If it's used for testing, can you put that?

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 3]
```diff
@@ -17,6 +17,7 @@
 package com.google.codeu.data;
 
 import com.google.appengine.api.datastore.DatastoreService;
+
```
Comment (kassidywebb) [inline src/main/java/com/google/codeu/data/Datastore.java]: Nit: delete space

**Review by kassidywebb — CHANGES_REQUESTED** [review]
Other than a few small changes, this is great! The comments on the datastore are very helpful. However, This is a very long pull request so for future pulls please try and make them smaller (like separating the testing feature from the comment feature, etc).

**Review by kassidywebb — APPROVED** [review]

File: .gitignore [hunk 4]
```diff
@@ -1,2 +1,6 @@
+
 /target/
 /.DS_Store/
+src/main/.DS_Store
```
Comment (ryanluo) [inline .gitignore]: you should be able to do something like *.DS_Store

File: src/main/java/com/google/codeu/data/Comment.java [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.codeu.data;
+
+import java.util.UUID;
+
+/** A single message posted by a user. */
```
Comment (ryanluo) [inline src/main/java/com/google/codeu/data/Comment.java]: Can you expand on this?

File: src/main/java/com/google/codeu/data/Comment.java [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+	private String imageUrl;
+	
+
+	/**
+	 * Constructs a new {@link Comment} posted by {@code user} with {@code text} content. Generates a
+	 * random ID and uses the current system time for the creation time. Added new imageUrl parameter
+	 * and set function.
+	 */
+
+	public Comment(){
```
Comment (ryanluo) [inline src/main/java/com/google/codeu/data/Comment.java]: What is this default constructor for?

**Review by ryanluo — APPROVED** [review]
Make a few changes and you're good to submit.

File: src/main/java/com/google/codeu/data/Comment.java (hunk 6 above)
Comment (Ajide49) [reply src/main/java/com/google/codeu/data/Comment.java]: It was copied over from the Event class on accident. I dont believe I need it.

**Pull request merged**